test/integration/gardenlet/seed/seed/seed_test.go

```diff
@@ -234,17 +248,19 @@ var _ = Describe("Seed controller tests", func() {
 						return secretList.Items
 					}).Should(HaveLen(1))
 
-					By("Verify that garden namespace was labeled and annotated appropriately")
-					Eventually(func(g Gomega) {
-						g.Expect(testClient.Get(ctx, client.ObjectKeyFromObject(testNamespace), testNamespace)).To(Succeed())
-						g.Expect(testNamespace.Labels).To(And(
-							HaveKeyWithValue("role", "garden"),
-							HaveKeyWithValue("high-availability-config.resources.gardener.cloud/consider", "true"),
-						))
-						g.Expect(testNamespace.Annotations).To(And(
-							HaveKeyWithValue("high-availability-config.resources.gardener.cloud/zones", "a,b,c"),
-						))
-					}).Should(Succeed())
+					if !seedIsGarden {
```

Comment:
Made this change, because https://github.com/gardener/gardener/blob/555bb9874d0507d2bd382e8a2657569f6a80e530/pkg/gardenlet/controller/seed/seed/reconciler_reconcile.go#L300-L304

Reply:
Great catch, thanks!